pylib/openrazer/client/devices/mice.py

```diff
@@ -97,6 +97,75 @@ def dpi(self, value: tuple):
         else:
             raise NotImplementedError()
 
+    @property
+    def dpi_stages(self) -> list:
+        """
+        Get mouse DPI stages
+
+        Will return a list of tuples
+        :return: DPI stages [(500, 500), (1000, 1000), (2000, 2000) ...]
+        :rtype: list
+
+        :raises NotImplementedError: if function is not supported
+        """
+        if self.has('dpi'):
+            raw_dpi_stages = self._dbus_interfaces['dpi'].getDPIStages()
+            dpi_stages = []
+
+            i = 0
+            while i + 1 < len(raw_dpi_stages):
+                dpi_x, dpi_y = raw_dpi_stages[i:i + 2]
+                i += 2
+
+                dpi_stages.append((int(dpi_x), int(dpi_y)))
+
+            return dpi_stages
+
+    @dpi_stages.setter
+    def dpi_stages(self, value: list):
+        """
+        Set mouse DPI stages
+
+        Daemon does type validation but can't be too careful
+        :param value: list of DPI X, Y tuple
+        :type value: list
+
+        :raises ValueError: If a tuple isn't long enough or contains invalid
+                            crap
+        :raises NotImplementedError: If function is not supported
+        """
+        if self.has('dpi'):
```

Comment:
this should get a new capabilities string like `dpi_stages`. These are registered in `pylib/openrazer/client/devices/__init__.py` (same for the getter above)

Reply:
Done in 8d2191ff00597557e8196a7def2f569d1cc2f7b6